src/components/redux/books/books-state.js

```diff
@@ -1,20 +1,20 @@
-const bookState = [{
-  key: '1',
-  categorie: 'Islamic',
-  title: 'Islamic Gold Era',
-  author: 'Maen khraisha',
-},
-{
-  key: '2',
-  categorie: 'Islamic',
-  title: 'Islamic Gold Era',
-  author: 'Maen khraisha',
-},
-{
-  key: '3',
-  categorie: 'Islamic',
-  title: 'Islamic Gold Era',
-  author: 'Maen khraisha',
-}];
+// const bookState = [{
+//   key: '1',
+//   categorie: 'Islamic',
+//   title: 'Islamic Gold Era',
+//   author: 'Maen khraisha',
+// },
+// {
+//   key: '2',
+//   categorie: 'Islamic',
+//   title: 'Islamic Gold Era',
+//   author: 'Maen khraisha',
+// },
+// {
+//   key: '3',
+//   categorie: 'Islamic',
+//   title: 'Islamic Gold Era',
+//   author: 'Maen khraisha',
+// }];
```

Comment:
- [ ] Let's avoid committing commented codes to our repo. If you require a previous version of your code, you can make use of git history. Commented codes make your codebase messy.